src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,39 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+You can import file types which aren't natively supported by Astro. There are many **Vite** and compatible **Rollup** plugins you can use. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of Vite Documentation.
+
+For example, the YAML data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin:
```

Comment:
Could be nice to add a subheading here — would also help people searching for YAML support.

```suggestion
### Example: YAML support

The YAML (`.yml`) data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin:
```